rest-api/etl/etl.sql

```diff
@@ -458,7 +509,7 @@ CREATE TABLE cdm.src_clean
     value_ppi_code              varchar(200),
     topic_value                 varchar(200),
     value_code_id               bigint,
-    value_number                decimal(20,6),
+    value_number                double,
```

Comment:
In Postgres, it looks like, observation.value_number in OMOP is  supposed to be a decimal:

https://github.com/OHDSI/CommonDataModel/blob/master/PostgreSQL/OMOP%20CDM%20ddl%20-%20PostgreSQL.sql#L490

So it seems like representing it as a double can lead to floating point issues. But in Oracle it's a FLOAT:

https://github.com/OHDSI/CommonDataModel/blob/master/Oracle/OMOP%20CDM%20ddl%20-%20Oracle.sql#L488

So... what is it supposed to be?

Reply:
In the OMOP specification, field value_as_number is declared as float.
https://github.com/OHDSI/CommonDataModel/wiki/OBSERVATION

But, in practice, it can be decimal or float. I agree that to avoid floating point issues it is a good approach to use decimal.

Reply:
@jmandel thoughts on this?